docs/winget.md

```diff
@@ -0,0 +1,53 @@
+# Installing Wassette with WinGet
+
+Wassette provides a WinGet package manifest for easy installation on Windows systems.
+
+## Installation
+
+Since WinGet doesn't support installing directly from URLs, you need to download the manifest first:
+
+```powershell
+# Download the manifest
+curl -o Microsoft.Wassette.yaml https://raw.githubusercontent.com/microsoft/wassette/main/winget/Microsoft.Wassette.yaml
```

Comment:
Recommend using `Invoke-WebRequest` instead of curl since it's for ppl using Windows.